core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeLexer.g4

```diff
@@ -1117,31 +1117,24 @@ VARIANT          : 'VARIANT';
 
 LISTAGG: 'LISTAGG';
 
-DUMMY: 'DUMMY'; //Dummy is not a keyword but rules reference it. As to be cleaned.
+DUMMY: 'DUMMY'; //Dummy is not a keyword but rules reference it in unfinished grammar - need to get rid
 
-SPACE: [ \t\r\n]+ -> channel(HIDDEN);
+SPACE: [ \t\r\n]+ -> skip;
 
 SQL_COMMENT    : '/*' (SQL_COMMENT | .)*? '*/' -> channel(HIDDEN);
 LINE_COMMENT   : '--' ~[\r\n]*                 -> channel(HIDDEN);
 LINE_COMMENT_2 : '//' ~[\r\n]*                 -> channel(HIDDEN);
 
 // TODO: ID can be not only Latin.
-DOUBLE_QUOTE_ID    : '"' ~'"'+ '"';
+DOUBLE_QUOTE_ID    : '"' ~[\r\n"]+ '"';
```

Comment:
This rule needs to be LL(2) in order to properly capture escaped double quotes (`""`).